src/pages/nfts/[address]/listings/new.tsx

```diff
@@ -3,22 +3,18 @@ import { useForm, Controller } from 'react-hook-form'
 import Link from 'next/link'
 import { useRouter } from 'next/router'
 import { useConnection, useWallet } from '@solana/wallet-adapter-react'
-import { AuctionHouseProgram } from '@metaplex-foundation/mpl-auction-house'
-import { MetadataProgram } from '@metaplex-foundation/mpl-token-metadata'
 import { NextPageContext } from 'next'
 import { map, prop, isEmpty, intersection, pipe, or, any, isNil } from 'ramda'
-import {
-  Transaction,
-  PublicKey,
-  LAMPORTS_PER_SOL,
-  SYSVAR_INSTRUCTIONS_PUBKEY,
-} from '@solana/web3.js'
+import { LAMPORTS_PER_SOL } from '@solana/web3.js'
 import { gql, useQuery } from '@apollo/client'
 import client from './../../../../client'
 import { NftLayout } from './../../../../layouts/Nft'
 import Button, { ButtonType } from './../../../../components/Button'
 import { Nft, Marketplace, GetNftData } from './../../../../types'
 import { toast } from 'react-toastify'
+import { ListingsClient } from '@holaplex/marketplace-js-sdk'
+import { Wallet } from '@metaplex/js'
+import { Nft as NftFromSdk } from '@holaplex/marketplace-js-sdk'
```

Comment:
Shouldn't double import here just grab all the required exprots from 1 import declaration.